pkg/manifests/types.go

```diff
@@ -73,6 +73,10 @@ type AlertmanagerMainConfig struct {
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 	// Defines resource requests and limits for the Alertmanager container.
 	Resources *v1.ResourceRequirements `json:"resources,omitempty"`
+	// Defines a list of Secrets in the same namespace as the Alertmanager object
+	// which shall be mounted into the Alertmanager Pods as volumes named secret-<secret-name>.
+	// and are mounted into /etc/alertmanager/secrets/<secret-name> in the ‘alertmanager’ container.
```

Comment:
How about we rephrase this a bit .. 

```suggestion
	// Defines a list of secrets that need to be mounted into the Alertmanager. 
	// The secrets must reside within the same namespace as the Alertmanager object. 
	// They will be added as volumes named secret-<secret-name> and mounted at
	// /etc/alertmanager/secrets/<secret-name> within the 'alertmanager' container of
	// the Alertmanager Pods.

```